src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -151,13 +151,14 @@ def _make_space(requested_size, current_build_dir=None):
   return result
 
 
-def _make_space_for_build(build_local_archive,
+def _make_space_for_build(archive_reader,
                           current_build_dir,
                           file_match_callback=None):
   """Make space for extracting the build archive by deleting the least recently
   used builds."""
-  extracted_size = archive.extracted_size(
-      build_local_archive, file_match_callback=file_match_callback)
+  extracted_size = 0
+  if archive_reader:
```

Comment:
So, this only existed because of `build_manager_test.py`, but I actually modified the latter to make this line not needed anymore, and forgot to remove it.